eng/pipelines/common/platform-matrix.yml

```
@@ -53,7 +53,7 @@ jobs:
         ${{ insert }}: ${{ parameters.jobParameters }}
 
 # Linux armv6
-- ${{ if or(containsValue(parameters.platforms, 'Linux_armv6'), in(parameters.platformGroup, 'all', 'gcstress')) }}:
+- ${{ if or(containsValue(parameters.platforms, 'Linux_armv6'), in(parameters.platformGroup, 'gcstress')) }}:
```

instead of doing this could we check for the mono runtimeFlavor explicitly:
```suggestion
- ${{ if or(containsValue(parameters.platforms, 'Linux_armv6'), or(and(eq(parameters.runtimeFlavor, 'mono'), in(parameters.platformGroup, 'gcstress')), and(ne(parameters.runtimeFlavor, 'mono'), in(parameters.platformGroup, 'all', 'gcstress'))) }}:
```
otherwise it seems pretty easy to miss in a future refactoring when comparing all the different platforms.